torch/nn/quantized/modules/conv.py

```diff
@@ -232,7 +232,8 @@ def from_float(cls, mod):
                 cls._FLOAT_MODULE.__name__ + " but got:" + str(type(mod))
             assert hasattr(mod, "qconfig"), \
                 "Input float module must have qconfig defined."
-            activation_post_process = mod.activation_post_process
+            activation_post_process = None if not hasattr(
```

Comment:
in get_qconv L206 in this file, can we check if activation_post_process is None or not, instead of using try/exception?
and we can also skip assigning scale and zero_point when activation_post_process is None as well

Reply:
no because the implementation has activation_post_process but it just raises an exception, will look at not assigning scale/zp

Reply:
added a scale/zp skip